mohawk/util.py

```diff
@@ -268,3 +267,37 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
+    """Returns a bewit identifier for the resource as a string"""
+    if resource.method != 'GET':
+        raise ValueError('bewits can only be generated for GET requests')
+    mac = calculate_mac(
+        'bewit',
+        resource,
+        None,
+    )
+
+    if isinstance(mac, six.binary_type):
+        mac = mac.decode('ascii')
+
+    if resource.ext is None:
+        ext = ''
+    else:
+        ext = resource.ext
+
+    # b64encode works only with bytes in python3, but all of our parameters are
+    # in unicode, so we need to encode them. The cleanest way to do this that
+    # works in both python 2 and 3 is to use string formatting to get a
+    # unicode string, and then explicitly encode it to bytes.
+    inner_bewit = u"{id}\\{exp}\\{mac}\\{ext}".format(
+        id=resource.credentials['id'],
+        exp=resource.timestamp,
+        mac=mac,
+        ext=ext,
```

Comment:
The Node server implementation [looks pretty strict](https://github.com/hueniverse/hawk/blob/master/lib/server.js#L364) so I think it would be a good idea to strip them or at least raise an exception.